ruby_on_rails-postgres/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
```

Comment:
Please use at least the `3.7`  Compose specification for the Compose file

Reply:
bug fixed